tests/unit/cmd/sca/test_scan.py

```diff
@@ -338,36 +342,6 @@ def test_sca_text_handler_ordering(patch_scan_all, cli_fs_runner):
 
     assert result.exit_code == ExitCode.SCAN_FOUND_PROBLEMS
     assert (
-        """> toto/Pipfile.lock: 2 incidents detected
-
->>> : Incident 1 (SCA): bar@2.5.6
-Severity: Critical
-Summary: 
-No fix is currently available.
-Identifier: 
-CVE IDs: -
-
->>> : Incident 2 (SCA): foo@1.0.5
-Severity: High
-Summary: 
-No fix is currently available.
-Identifier: 
-CVE IDs: -
-
-> Pipfile.lock: 2 incidents detected
-
->>> : Incident 1 (SCA): titi@2.0.0
-Severity: Medium
-Summary: 
-No fix is currently available.
-Identifier: 
-CVE IDs: -
-
->>> : Incident 2 (SCA): toto@1.0.0
-Severity: Low
-Summary: 
-No fix is currently available.
-Identifier: 
-CVE IDs: -"""  # noqa W291
+        """> toto/Pipfile.lock: 2 incidents detected\n\n>>> : Incident 1 (SCA): bar@2.5.6\nSeverity: Critical\nSummary: \nNo fix is currently available.\nIdentifier: \nCVE IDs: -\n\n>>> : Incident 2 (SCA): foo@1.0.5\nSeverity: High\nSummary: \nNo fix is currently available.\nIdentifier: \nCVE IDs: -\n\n> Pipfile.lock: 2 incidents detected\n\n>>> : Incident 1 (SCA): titi@2.0.0\nSeverity: Medium\nSummary: \nNo fix is currently available.\nIdentifier: \nCVE IDs: -\n\n>>> : Incident 2 (SCA): toto@1.0.0\nSeverity: Low\nSummary: \nNo fix is currently available.\nIdentifier: \nCVE IDs: -"""  # noqa W291
```

Comment:
Why modifying this ? It was more readable in a multi-line string no ?

Reply:
It's a nightmare to keep the old versions with unnecessary spaces, tried to revert but it completely broke my PR, re-starting again